### ISSUE TITLE
escalation: scan jobs with thousands of nodes working but reporting blank node errors

### ISSUE THREAD

**Issue by vjeffrey** [chef/automate#2252]
## Describe the bug
Sorry if the title is confusing.
tl;dr customer is running scan jobs on about 2000 nodes.
reports are being generated and ingested for most of them, as expected.
but the scan job lists all the nodes as having failed, with no error message.

_original issues: https://chefio.atlassian.net/browse/TRI-901 https://getchef.zendesk.com/agent/tickets/23215_

## To Reproduce
- load lots of nodes, most reachable, a few unreachable
- create a scan job to scan all the nodes
- see all nodes reported as failed in the scan jobs view despite having ingested reports from the nodes

_please note: i have not been able to reproduce this yet_

## Expected behavior
Only the nodes that we could not reach marked as failed, with an error message.

## Screenshots
<img width="949" alt="Screen Shot 2019-11-18 at 14 37 16" src="https://user-images.githubusercontent.com/10341541/69096064-f25a9a80-0a10-11ea-9441-541e4fb76798.png">

## Versions (please complete the following information):
- Automate Build Number: 20190904132002

**Comment by vjeffrey** [comment]
medium t shirt size, but unknown b/c no repro yet

**Comment by vjeffrey** [comment]
closing as the issue is not reproducible and we believe the merged pr fixes will help with the lack of errors the user was seeing
